contracts/TokenHolderRevenueFund.sol

```diff
@@ -154,7 +166,7 @@ contract TokenHolderRevenueFund is Ownable, AccrualBeneficiary, Servable, Transf
         uint256 currencyId, string memory standard)
     public
     {
-        require(amount.isNonZeroPositiveInt256(), "Amount not strictly positive [TokenHolderRevenueFund.sol:157]");
+        require(amount.isNonZeroPositiveInt256(), "Amount not strictly positive [TokenHolderRevenueFund.sol:169]");
```

Comment:
There exists a pre-processor package `solpp` which has the build in line number macro  '__line' which might be of interest. Not sure how stable the package is though.

Reply:
I'll look into it. Thanks.